src/util/find_macros.h

```diff
@@ -1,27 +0,0 @@
-/*******************************************************************\
```

Comment:
`find_macros.h` is used in EBMC

Reply:
Moved to #2213 to receive your separate approval/further discussion.